config/endpoint/endpoint_test.go

```diff
@@ -5,6 +5,7 @@ import (
 	"crypto/tls"
 	"crypto/x509"
 	"errors"
+	"github.com/TwiN/gatus/v5/config/endpoint/mqtt"
```

Comment:
This should be separate from the standard library dependencies, namely, at the bottom. See other go files for example.

Reply:
👍

Reply:
Let me know/ping me when you resolved it 👀

Reply:
Any news?